lib/pricegrabber/request.rb

```diff
@@ -66,39 +60,28 @@ def version
     end
 
     def pid
```

Comment:
This method could probably go away, unless it is being used from somewhere else?